loader/src/main/java/net/neoforged/fml/common/asm/RuntimeDistCleaner.java

```diff
@@ -128,22 +127,18 @@ private static List<AnnotationNode> unpack(final List<AnnotationNode> anns) {
         List<AnnotationNode> ret = anns.stream().filter(ann -> Objects.equals(ann.desc, ONLYIN)).collect(Collectors.toList());
         anns.stream().filter(ann -> Objects.equals(ann.desc, ONLYINS) && ann.values != null)
                 .map(ann -> (List<AnnotationNode>) ann.values.get(ann.values.indexOf("value") + 1))
-                .filter(v -> v != null)
-                .forEach(v -> v.forEach(ret::add));
+                .filter(Objects::nonNull)
+                .forEach(ret::addAll);
         return ret;
     }
 
     private boolean remove(final List<AnnotationNode> anns, final String side) {
-        return unpack(anns).stream().filter(ann -> Objects.equals(ann.desc, ONLYIN)).filter(ann -> ann.values.indexOf("_interface") == -1).anyMatch(ann -> !Objects.equals(((String[]) ann.values.get(ann.values.indexOf("value") + 1))[1], side));
+        return unpack(anns).stream().filter(ann -> Objects.equals(ann.desc, ONLYIN)).filter(ann -> !ann.values.contains("_interface")).anyMatch(ann -> !Objects.equals(((String[]) ann.values.get(ann.values.indexOf("value") + 1))[1], side));
     }
 
-    @SuppressWarnings("unchecked")
-    @Override
-    public Consumer<Dist> getExtension() {
-        return (s) -> {
-            DIST = s.name();
-            LOGGER.debug(DISTXFORM, "Configuring for Dist {}", DIST);
-        };
+    public static void setDistribution(Dist dist) {
```

Comment:
This method should either be non-static or be called statically instead of on an instance of this class